models/brats_mri_segmentation/configs/multi_gpu_train.json

```diff
@@ -1,6 +1,6 @@
 {
     "local_rank": "$os.environ['LOCAL_RANK']",
-    "device": "$torch.device(f'cuda:{@local_rank}')",
```

Comment:
Hi @Nic-Ma , I met an error when using this format. I'm not sure if it's an issue of "f"

Reply:
It's an issue to use "f" in an expression. For example, if define:
```
    "cuda_num": "0",
    "test_device": "$f'cuda:{@cuda_num}.'",
```
will see an error like:
```
  File "/opt/monai/monai/bundle/config_parser.py", line 291, in get_parsed_content
    return self.ref_resolver.get_resolved_content(id=id, **kwargs)
  File "/opt/monai/monai/bundle/reference_resolver.py", line 190, in get_resolved_content
    return self._resolve_one_item(id=id, **kwargs)
  File "/opt/monai/monai/bundle/reference_resolver.py", line 160, in _resolve_one_item
    self._resolve_one_item(id=d, waiting_list=waiting_list, **kwargs)
  File "/opt/monai/monai/bundle/reference_resolver.py", line 160, in _resolve_one_item
    self._resolve_one_item(id=d, waiting_list=waiting_list, **kwargs)
  File "/opt/monai/monai/bundle/reference_resolver.py", line 172, in _resolve_one_item
    item.evaluate(globals={f"{self._vars}": self.resolved_content}) if run_eval else item
  File "/opt/monai/monai/bundle/config_item.py", line 363, in evaluate
    optional_module = self._parse_import_string(value[len(self.prefix) :])
  File "/opt/monai/monai/bundle/config_item.py", line 333, in _parse_import_string
    node = first(ast.iter_child_nodes(ast.parse(import_string)))
  File "/usr/lib/python3.8/ast.py", line 47, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 1
    f'cuda:{__local_refs['cuda_num']}.'
```

Reply:
Let me investigate it.

Thanks.